src/infra/core/template.yaml

```diff
@@ -56,6 +60,54 @@ Resources:
         - Key: Source
           Value: !Ref SourceTagValue
 
+  DynamoDBEncryptionKey:
```

Comment:
I think let's make this specific to the AccountStatusTable DynamoDB table instead of being a generic KMS key for tables, I know we have the single table but rather make it more explicit.

Can see it's restricted to root and AccountStatusTable but may be worth updating the resource name, description and alias name